migtests/tests/pg/basic-public-live-test/validateAfterChanges

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
```

Comment:
Change one of the non-public test to this public one to have at least one test for both of these in GH.